kernel/configs/x86_64_kata_kvm_4.14.x

```diff
@@ -448,6 +448,7 @@ CONFIG_CPU_SUP_INTEL=y
 # CONFIG_CPU_SUP_AMD is not set
 # CONFIG_CPU_SUP_CENTAUR is not set
 CONFIG_HPET_TIMER=y
+CONFIG_HPET_EMULATE_RTC=y
```

Comment:
Why is this one needed? Are we expecting the kernel to emulate an RTC based on HPET?

Reply:
It comes when I selected `CONFIG_RTC_CLASS`. Looking closer, `CONFIG_RTC_DRV_CMOS` and `HPET_TIMER` automatically enables this feature.

Reply:
I'm concerned that having those enabled might fail the kernel from booting properly if they do expect HPET or CMOS to be emulated, which is not the case for NEMU.